Firestore/core/src/firebase/firestore/local/leveldb_key.h

```diff
@@ -142,7 +143,7 @@ class LevelDbMutationKey {
  private:
   // Deliberately uninitialized: will be assigned in Decode
```

Comment:
Looks like the comment is now stale? (here and below)

Reply:
Agreed.